src/privatesend/privatesend-server.cpp

```diff
@@ -433,9 +434,9 @@ void CPrivateSendServer::ChargeFees(CConnman& connman)
     if (nState == POOL_STATE_ACCEPTING_ENTRIES) {
         for (const auto& txCollateral : vecSessionCollaterals) {
             bool fFound = false;
-            for (const auto& entry : vecEntries)
-                if (*entry.txCollateral == *txCollateral)
-                    fFound = true;
+            for (const auto& entry : vecEntries) {
+                if (*entry.txCollateral == *txCollateral) fFound = true;
```

Comment:
Actually, we could also fix this while at it
```suggestion
                if (*entry.txCollateral == *txCollateral) {
                    fFound = true;
                    break;
                }
```